mlrun/api/api/endpoints/model_endpoints.py

```diff
@@ -188,7 +187,6 @@ async def patch_model_endpoint(
 
 @router.delete(
     "/projects/{project}/model-endpoints/{endpoint_id}",
-    status_code=HTTPStatus.NO_CONTENT.value,
```

Comment:
No need to define `status_code` in the API decorator in this case. By default it will return `ok` (200). Not sure why we have defined `HTTPStatus.NO_CONTENT.value` in the 1st place, perhaps @Tankilevitch has more insights about it.